File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/classes/PhoneStateCallback.java
package com.atomykcoder.atomykplay.classes;

import android.os.Build;
import android.telephony.TelephonyCallback;

import androidx.annotation.RequiresApi;

@RequiresApi(api = Build.VERSION_CODES.S)
public class PhoneStateCallback extends TelephonyCallback implements TelephonyCallback.CallStateListener {
    @Override
    public void onCallStateChanged(int state) {

    }
}

File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/classes/PhoneStateCallback.kt
package com.atomykcoder.atomykplay.classes

import android.os.Build
import android.telephony.TelephonyCallback
import android.telephony.TelephonyCallback.CallStateListener
import androidx.annotation.RequiresApi

@RequiresApi(api = Build.VERSION_CODES.S)
open class PhoneStateCallback : TelephonyCallback(), CallStateListener {
    override fun onCallStateChanged(state: Int) {}
}
